GitHubActions/execute_test.sh

```diff
@@ -22,7 +22,7 @@
 VERSION=$1
 cd ./workdir/postgresql-${VERSION}/contrib/sqlite_fdw
 chmod +x ./test.sh
-./test.sh
+./test.sh "${@:2}"
```

Comment:
It belongs to [build_sqlite_fdw.sh](https://github.com/pgspider/sqlite_fdw/pull/103/files#diff-943d09e74c7d5e1fc387379e2528cdda7924bc6da51b124c10759f525e9b640a) step. Please refer line: https://github.com/pgspider/sqlite_fdw/pull/103/files#diff-3ab46ee209a127470fce3c2cf106b1a1dbadbb929a4b5b13656a4bc4ce19c0b8R73

Reply:
Thanks, @MinhLA1410. So, should I change `Makefile`? I think in case of `make CFLAGS=-DGIS_SUPPORT=1` some original `CFLAGS` can be lost. Do you know some more safe method of _define_ passing?

Reply:
I have a variant with a special variable 
```make
ifdef GIS
override CFLAGS := $(CFLAGS) -DGIS_SUPPORT=1
endif
```
and `make GIS=1` or `GIS=1 make`. Will this normal?

Reply:
It's normal. No problem.

Reply:
Thanks, @MinhLA1410 ! This is similar case. I'll adopt my GIS PR to your PR in some days. Also I am waiting on merging of this PR.